setup.py

```diff
@@ -0,0 +1,17 @@
+import setuptools
+
+# install package with 'python3 setup.py install'
+
+setuptools.setup(
+    name="gen_src",
+    version="2",
+    author="Woohyun Han",
```

Comment:
@SeonghaEom Please feel free to add you here XD

Reply:
You are angel + genius.. 😊